dotnet/src/AutoGen.Core/Extension/GroupChatExtension.cs

```diff
@@ -51,7 +51,10 @@ public static void AddInitializeMessage(this IAgent agent, string message, IGrou
                 yield break;
             }
 
-            chatHistory = messages;
+            // messages will contain the complete chat history, include initalize messages
+            // but we only need to add the last message to the chat history
+            // fix #3268
```

Comment:
Nit : do we want to have issue numbers in the source code ?

Reply:
I think it's fine, the issue number can provide more context.